### PR TITLE
Issue/fix terminated instance dropdown

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes *Add ticket reference here*

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

~~- [] Attached issue to pull request~~
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
- [ ] End user documentation is included or an issu

https://github.com/inmanta/web-console/assets/113331659/c4254ea2-c341-4d6b-96e9-f5ea31690023

e is created for end-user documentation (add ref to issue here: )

### CODE REVIEW

File: changelogs/unreleased/terminated-instance-dropdown-fix.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+description: "Fix issue with misleading visual state of Diagnose button and misplaced 'back' button for terminated instances an"
```
Comment (LukasStordeur) [inline changelogs/unreleased/terminated-instance-dropdown-fix.yml:1]: an?

File: src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx [hunk 2]
```diff
@@ -285,7 +292,13 @@ export const RowActions: React.FunctionComponent<InstanceActionsProps> = ({
       </MenuContent>
     </Menu>
   );
-
+  console.log(
```
Comment (LukasStordeur) [inline src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx:295]: console.log

Comment (matborowczyk) [reply src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx]: I thought I removed that, I guess I did only on other branch

File: changelogs/unreleased/terminated-instance-dropdown-fix.yml (hunk 1 above)
Comment (matborowczyk) [reply changelogs/unreleased/terminated-instance-dropdown-fix.yml]: removed

**Review by LukasStordeur — APPROVED** [review]

**Pull request closed without merging**